drivers/volume/aws/aws.go

```diff
@@ -523,6 +523,14 @@ func (a *aws) GetRestoreStatus(restore *storkapi.ApplicationRestore) ([]*storkap
 		}
 		ebsVolume, err := a.getEBSVolume(vInfo.RestoreVolume, nil)
 		if err != nil {
+			if awsErr, ok := err.(awserr.Error); ok {
+				if awsErr.Code() == "InvalidVolume.NotFound" {
+					vInfo.Status = storkapi.ApplicationRestoreStatusFailed
+					vInfo.Reason = "Restore failed for volume: NotFound"
+					continue
```

Comment:
You can't continue, this will not return info for this volume, need to append vInfo to volumeInfos

Reply:
Fixed. Appending vInfo to VolumeInfos, before continue.